### PR TITLE
feat(cli): improve appeal help messages

### PR DESCRIPTION
**Pull request by ravisuhag**
- Fix usage guide on the wrong command. 
- Improve appeal help messages.

### CODE REVIEW

File: cmd/appeal.go [hunk 1]
```diff
@@ -260,9 +270,10 @@ func rejectApprovalStepCommand(c *app.CLIConfig) *cobra.Command {
 		},
 	}
 
-	cmd.Flags().UintVar(&id, "id", 0, "appeal id")
+	cmd.Flags().UintVar(&id, "id", 0, "ID of the appeal")
 	cmd.MarkFlagRequired("id")
-	cmd.Flags().StringVarP(&approvalName, "approval-name", "a", "", "approval name going to be approved")
+
+	cmd.Flags().StringVarP(&approvalName, "approval-name", "a", "", "Approval name going to be approved")
```
Comment (ravisuhag) [inline cmd/appeal.go:276]: @rahmatrhd approval-name is the name of the approval step or the name of the person who is approving?

Comment (rahmatrhd) [reply cmd/appeal.go:281]: name of the approval step

Comment (ravisuhag) [reply cmd/appeal.go:281]: Then lets call the flag step?  `approval-name` is bit confusing. WDYT?

Comment (rahmatrhd) [reply cmd/appeal.go:281]: yeah, that sounds better 👍

**Review by rahmatrhd — APPROVED** [review]

File: cmd/appeal.go (hunk 1 above)
Comment (ravisuhag) [reply cmd/appeal.go]: @rahmatrhd done. Please check now.

**Pull request merged**